### ISSUE TITLE
Recent import changes make it difficult to override variables

### ISSUE THREAD

**Issue by mzgajner** [jgthms/bulma#323]
### Checklist

* [x] This is about the Bulma **CSS framework**
* [x] I'm using Bulma **version** 0.2.1
* [x] I am sure this issue is **not a duplicate**

### Description

Before recent changes, Bulma imported variables [in just one place](https://github.com/jgthms/bulma/blob/0.1.2/sass/utilities/utilities.sass#L8) and all the modules assumed they were available, a technique [Bootstrap](https://github.com/twbs/bootstrap/blob/v4-dev/scss/bootstrap.scss#L10) and similar libraries use. This had a handy consequence - you could import Bulma files to your own project directly from `node_modules` and if you added a custom `variables.scss` import in the right place, Bulma modules received your custom values. This is how my `style.scss` looked before Bulma 0.2:

```scss
@import "bulma/sass/utilities/utilities";
@import "variables"; // Local file that overrides some variables
@import "bulma/sass/base/base";
@import "bulma/sass/elements/elements";
@import "bulma/sass/components/components";
@import "bulma/sass/layout/layout";
```

But after [this commit](https://github.com/jgthms/bulma/commit/70b136705367f61943e8adea75e6c98c0857b532), every file that needs variables imports them locally. This means we don't have an elegant solution for overriding variables without copying Bulma files into our project, which is generally a bad practice.

**Comment by jeffijoe** [comment]
Was just about to open the same issue.

**Comment by mzgajner** [comment]
Fixed in 0.2.3, tnx! 👍
